devicon.json

```
@@ -3044,6 +3044,16 @@
         "color": "#494949",
         "aliases": []
     },
+    {
+        "name": "xd",
```

Choosing `xd` as the name of the icon could be missleading. I would suggest to name it `adobexd`. @Thomas-Boi What do you think? (I'm thinking of `aftereffects` and `premierepro` where we have no `adobe` in front).

Please have a look at this discussion https://github.com/devicons/devicon/discussions/459, we decided to not introduce hyphens.

```suggestion
        "name": "adobexd",
```

---

At first, I choose `xd` becase all other Adobe products in this icon set don't have the `adobe` prefix to keep the consistancy. But just `xd` can be confused as well. So I don't have a string opinion on this 🤔